config/assets/default.js

```diff
@@ -34,7 +34,7 @@ module.exports = {
       'modules/*/client/*.js',
       'modules/*/client/**/*.js'
     ],
-    views: ['modules/*/client/views/**/*.html'],
+    views: ['modules/*/client/**/*client.view.html'],
```

Comment:
I guess you're right... maybe 'modules/*/client/**/*.html' like is done for 'modules/*/client/**/*.js' would be the most flexible one (and wouldn't break compatibility with code coming from an earlier version)